supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,95 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+=== Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Do not use JavaScript links.
++
+Example JavaScript:
```

Comment:
Can you use the proper `.Example JavaScript` title?

It also needs a `+` under it so that the code block is indented